snippets/freeipa_register.erb

```diff
@@ -24,6 +24,9 @@ name: freeipa_register
 #
 #   freeipa_opts                Additional options to pass directly to installer
 #
+
+/usr/sbin/sshd-keygen
```

Comment:
This only appears to exist on EL7 and higher, not EL6 or Debian/Ubuntu, which this snippet probably supports. Can you add equivalents?